packages/client/src/submodules/profile/Profile.ts

```diff
@@ -207,16 +207,11 @@ export class Profile {
     });
   }
 
-  async create(
+  async createWithHandle(
     request: CreateProfileWithHandleRequest,
-  ): PromiseResult<
-    RelaySuccessFragment | CreateProfileWithHandleErrorResultFragment,
-    CredentialsExpiredError | NotAuthenticatedError
-  > {
-    return requireAuthHeaders(this.authentication, async (headers) => {
-      const result = await this.sdk.CreateProfileWithHandle({ request }, headers);
-      return result.data.result;
-    });
+  ): PromiseResult<RelaySuccessFragment | CreateProfileWithHandleErrorResultFragment, never> {
+    const result = await this.sdk.CreateProfileWithHandle({ request });
+    return success(result.data.result);
```

Comment:
if the result is always a success, then we don't need to wrap it in the Result at all, do we? I'd rather change the return type then.

Reply:
so `Promise<RelaySuccessFragment | CreateProfileWithHandleErrorResultFragment>` ??

Reply:
sure!